packages/amplify-graphql-transformer-migrator/src/utils.ts

```diff
@@ -1,4 +1,5 @@
-import { $TSObject, AmplifySupportedService, stateManager } from '@aws-amplify/amplify-cli-core';
+import { AmplifySupportedService, stateManager } from '@aws-amplify/amplify-cli-core';
```

Comment:
This `AmplifySupportedService` looks like an easy lift here? Maybe we can move it to interfaces or core?

Reply:
I'll take a look.